health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/HypothesisWrapper.js

```
@@ -1,425 +1,390 @@
+  const { state, dispatch } = useMyContext();
+  const [assumptionValues, setAssumptionValues] = useState(
+    Digit.SessionStorage.get("MICROPLAN_DATA")?.HYPOTHESIS?.Assumptions?.assumptionValues || []
+  );
+  const assumptionsFormValues = customProps?.sessionData?.ASSUMPTIONS_FORM?.assumptionsForm; //array with key and value
+  const campaignType = customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.campaignType?.code;
+  const resourceDistributionStrategyCode =
+    customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.distributionStrat?.resourceDistributionStrategyCode;
+  const searchParams = new URLSearchParams(window.location.search);
+  const [internalKey, setInternalKey] = useState(() => {
+    const keyParam = searchParams.get("internalKey");
+    return keyParam ? parseInt(keyParam) : 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)